SCClassLibrary/Common/Collections/Event.sc

```diff
@@ -44,17 +47,9 @@ Event : Environment {
 		//		^this.delta
 	}
 
-	// this[\isRest] may be nil
 	isRest {
-		^this[\isRest] == true
-		or: { this[\type] == \rest
-			or: {
-				this.use {
-					parent ?? { parent = defaultParentEvent };
-					~detunedFreq.value.isRest
-				}
-			}
-		}
+		_Event_IsRest
+		^false
```

Comment:
Just checked it to make sure I was understanding everything correctly. Before, there used to be some actual code after the primitive; now, just ^false. Is there a reason for keeping ^false as an SC-level fallback? It would be much faster and cleaner to put it in the primitive.